sdk/push/build.gradle

```diff
@@ -0,0 +1,6 @@
+project.description = 'Push functionality for the Klaviyo SDK suite'
+evaluationDependsOn(':sdk')
+
+dependencies {
+    implementation project(':sdk:core')
```

Comment:
Our push module will have a dependency on our core SDK so I'm gonna declare that here